webservices/tasks/__init__.py

```diff
@@ -45,8 +46,17 @@ def redis_url():
         'webservices.tasks.legal_docs',
     ),
     CELERYBEAT_SCHEDULE=schedule,
+    acks_late=True,
 )
 
+app.conf.ONCE = {
+    'backend': 'celery_once.backends.Redis',
+    'settings': {
+        'url': redis_url(),
+        'default_timeout': 60 * 60
+    }
+}
```

Comment:
That was because of a syntax change in upgrading celery once, I read the docs a while ago so I am not remembering the details.